### PR TITLE
File does not to be writable here.

### LINKED ISSUE #1345: Why deps.json and dev.json need to be writable?

### PR DESCRIPTION
**Pull request by qingjuntian**
## Description
Please add a meaningful description for this change.
Ensure the PR has required unit tests.

## Related issue
Kindly link any related issues. E.g. Fixes #1345 .

### CODE REVIEW

File: src/Microsoft.TestPlatform.CoreUtilities/Helpers/FileHelper.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public bool DirectoryExists(string path)
         }
 
         /// <inheritdoc/>
-        public Stream GetStream(string filePath, FileMode mode, FileAccess access = FileAccess.ReadWrite)
+        public Stream GetStream(string filePath, FileMode mode, FileAccess access = FileAccess.Read)
```
Comment (smadala) [inline src/Microsoft.TestPlatform.CoreUtilities/Helpers/FileHelper.cs]: This function is used in other places too, Not just in DotnetTestHostManager.cs. Pass `access: FileAccess.Read` at [here](https://github.com/Microsoft/vstest/blob/master/src/Microsoft.TestPlatform.TestHostProvider/Hosting/DotnetTestHostManager.cs#L368) and [here](https://github.com/Microsoft/vstest/blob/master/src/Microsoft.TestPlatform.TestHostProvider/Hosting/DotnetTestHostManager.cs#L394)

**Review by smadala — DISMISSED** [review]

File: src/Microsoft.TestPlatform.CoreUtilities/Helpers/FileHelper.cs (hunk 1 above)
Comment (qingjuntian) [reply src/Microsoft.TestPlatform.CoreUtilities/Helpers/FileHelper.cs]: @smadala the default parameter is FileAccess.Read with this PR right? Why do we need to pass that in this way?

Comment (smadala) [reply src/Microsoft.TestPlatform.CoreUtilities/Helpers/FileHelper.cs]: We can't change the default value from ReadWrite to Read, Without changing it's references. I suggest we keep default value ReadWrite( `Opens a FileStream on the specified path with read/write access.` [More details](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.open?view=netframework-4.7.1#System_IO_File_Open_System_String_System_IO_FileMode_)). And just pass the access value Read in DotnetTestHostManager.cs

**Comment by smadala** [comment]
@qingjuntian I have made changes need, Can you please sign [CLA](https://cla.opensource.microsoft.com/Microsoft/vstest?pullRequest=1372)? So that we can merge the PR once all checks passed.

**Comment by qingjuntian** [comment]
thanks @smadala

**Review by mayankbansal018 — APPROVED** [review]

**Pull request merged**